migration/CMTOOL-284_JBoss_EAP_7.4_Server_Migration_Tool.adoc

```diff
@@ -0,0 +1,160 @@
+** add default configurations for JBoss EAP 7.4 subsystems, which didn't exist in JBoss EAP 7.0, and are required for the server to work correctly
+** subsystem configuration updates necessary for the server configurations to work as expected with JBoss EAP 7.4
+** migration of resources, which paths are referenced by migrated server configurations, such as property files, keystores and certificates
+** migration of modules referenced by the server configurations
+
+* The migration of JBoss EAP 7.1 server configurations includes:
+** add default configurations for JBoss EAP 7.4 subsystems, which didn't exist in JBoss EAP 7.1, and are required for the server to work correctly
+** subsystem configuration updates necessary for the server configurations to work as expected with JBoss EAP 7.4
+** migration of resources, which paths are referenced by migrated server configurations, such as property files, keystores and certificates
+** migration of modules referenced by the server configurations
+
+* The migration of JBoss EAP 7.2 server configurations includes:
+** add default configurations for JBoss EAP 7.4 subsystems, which didn't exist in JBoss EAP 7.2, and are required for the server to work correctly
+** subsystem configuration updates necessary for the server configurations to work as expected with JBoss EAP 7.4
+** migration of resources, which paths are referenced by migrated server configurations, such as property files, keystores and certificates
+** migration of modules referenced by the server configurations
+
+=== Non-Requirements
+
+* Migration of deployments found on source server does not includes any required or optional modifications to its content.
+* Migration should not include the runtime related data present in the source server.
+
+== Implementation Plan
+
+* The migration tool for JBoss EAP 7.4 is the JBoss Server Migration Tool, which is developed as a standalone project:
+** Source repo and community binaries at https://github.com/wildfly/wildfly-server-migration[GitHub]
+** Project management at https://issues.jboss.org/projects/CMTOOL[JIRA]
+
+* The tool is not a standalone product, it’s simply a component in the JBoss EAP product, and should follow same procedures established for other JBoss EAP components with respect to Productization, QE and Support. The MIGRATION component in WFLY JIRA should be used for issues related with the integration of the tool in the JBoss EAP 7.4 distributions.
+
+* The “productized” tool should be delivered in the JBoss EAP 7.4 server distribution, by provisioning a Galleon Feature Pack specific for it, adding the following content:
+** a readme with basic info and instructions at `/migration`
+** the tool configuration files at `/migration/configuration`
+** the sh and bat scripts (jboss-server-migration.*) to start the tool at `/bin`
+** a system module with the tool’s artifacts/resources at `/modules/system/base/org/jboss/migration/cli`
+
+== Test Plan
+
+* TODO
```

Comment:
Can you update this part please?

Reply:
in the past we updated this later with the polarion link, do you want to do it different this time?

Reply:
Since this is publicly available, there should be no links to internal systems. I suppose a note that there will be no testing and no new tests for upstream and the testing is ensured just by QE should be enough.

Reply:
done